google-required-permissions.html.md.erb

```diff
@@ -0,0 +1,172 @@
+    description: Allows BOSH Google CPI to perform all BOSH Director actions
+    name: projects/((project_id))/roles/bosh.director
+    included_permissions:
+    # addresses
+    - compute.addresses.get
+    - compute.addresses.list
+
+    # backend services
+    - compute.backendServices.get
+    - compute.backendServices.list
+
+    # disk types
+    - compute.diskTypes.get
+
+    # disks
+    - compute.disks.delete
+    - compute.disks.list
+    - compute.disks.get
+    - compute.disks.createSnapshot
+    - compute.snapshots.create
+    - compute.disks.create
+    - compute.images.useReadOnly
+
+    # global operations
+    - compute.globalOperations.get
+    # images
+    - compute.images.delete
+    - compute.images.get
+    - compute.images.create
+
+    # instance groups
+    - compute.instanceGroups.get
+    - compute.instanceGroups.list
+    - compute.instanceGroups.update
+
+    # instances
+    - compute.instances.setMetadata
+    - compute.instances.setLabels
+    - compute.instances.setTags
+    - compute.instances.reset
+    - compute.instances.start
+    - compute.instances.list
+    - compute.instances.get
+    - compute.instances.delete
+    - compute.instances.create
+    - compute.subnetworks.use
+    - compute.subnetworks.useExternalIp
+    - compute.instances.detachDisk
+    - compute.instances.attachDisk
+    - compute.disks.use
+    - compute.instances.deleteAccessConfig
+    - compute.instances.addAccessConfig
+    - compute.addresses.use
+
+    # machine type
+    - compute.machineTypes.get
+
+    # region operations
+    - compute.regionOperations.get
+
+    # zone operations
+    - compute.zoneOperations.get
+
+    # networks
+    - compute.networks.get
+
+    # subnetworks
+    - compute.subnetworks.get
+
+    # snapshots
+    - compute.snapshots.delete
+    - compute.snapshots.get
+
+    # target pool
+    - compute.targetPools.list
+    - compute.targetPools.get
+    - compute.targetPools.addInstance
+    - compute.targetPools.removeInstance
+    - compute.instances.use
+
+    # Storage services - used when uploading heavy stemcells
+    - storage.buckets.create
+    - storage.objects.create
+    ```
+
+3. Create the BOSH Director role:
+
+    ```bash
+    gcloud beta iam roles --project <project-id> create bosh.director \
+      --file <( bosh int -v project_id=<project-id> bosh-director-role.yml )
+    ```
+
+4. On Google Cloud navigate go to `IAM & admin` > `Service accounts` and click on `CREATE SERVICE ACCOUNT`.
+   Give your service account a name, check `Furnish a new private key` and select the aforementioned role.
+
+   <%= image_tag("images/gcp-service-account.png") %>
+
+---
+## <a id="director-with-gcs-blobstore"></a> Google CPI and Director configured with a GCS blobstore
+
+This configuration is similar to the previous one except that it's used when the Director and the Agents use GCS as their [blobstore](bosh-components.html#blobstore) instead of an internal blobstore provided by the bosh release.
```

Comment:
Might be useful to clarify these are used in addition to the previous one. The "... is similar to the previous one ..." part makes it sound a little like this is an alternative to that.